src/api/loader.coffee

```diff
@@ -41,22 +41,27 @@ getResponseDataByEnv = (isLocal, eventData, response) ->
 
 
 handleAPIEvent = (apiEventChannel, baseUrl, setting, eventData = {}) ->
+
   isLocal = setting.loadLocally
   # simulate server delay
   delay = if isLocal then 200 else 0
 
   apiSetting = getAjaxSettingsByEnv(isLocal, baseUrl, setting, eventData)
+  if apiSetting.method is 'GET'
```

Comment:
mmhm, yeah this is awesome for now, but i think it'd be nice to make it more robust later on